<ui version="4.0" >
 <class>order</class>
 <widget class="QWidget" name="order" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>409</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize" >
   <size>
    <width>535</width>
    <height>409</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>535</width>
    <height>409</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>OrderList</string>
  </property>
  <widget class="QTableView" name="table_order" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>70</y>
     <width>450</width>
     <height>290</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label2" >
   <property name="geometry" >
    <rect>
     <x>173</x>
     <y>380</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>Total Need to Pay : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label1" >
   <property name="geometry" >
    <rect>
     <x>40</x>
     <y>40</y>
     <width>61</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string>Order ID : </string>
   </property>
  </widget>
  <widget class="QLabel" name="OrderIDLabel" >
   <property name="geometry" >
    <rect>
     <x>120</x>
     <y>40</y>
     <width>131</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PayLabel" >
   <property name="geometry" >
    <rect>
     <x>300</x>
     <y>380</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label3" >
   <property name="geometry" >
    <rect>
     <x>370</x>
     <y>380</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text" >
    <string>RMB</string>
   </property>
  </widget>
  <widget class="QLabel" name="label1_2" >
   <property name="geometry" >
    <rect>
     <x>130</x>
     <y>10</y>
     <width>271</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text" >
    <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:12pt; font-weight:600; color:#00008b;">Below is Your Order Imformation~&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
